drivers/docker/config.go

```diff
@@ -242,10 +243,21 @@ var (
 		})),
 		"mac_address": hclspec.NewAttr("mac_address", "string", false),
 		"mounts": hclspec.NewBlockSet("mounts", hclspec.NewObject(map[string]*hclspec.Spec{
+			"type": hclspec.NewDefault(
+				hclspec.NewAttr("type", "string", false),
+				hclspec.NewLiteral("\"volume\""),
+			),
 			"target":   hclspec.NewAttr("target", "string", false),
 			"source":   hclspec.NewAttr("source", "string", false),
 			"readonly": hclspec.NewAttr("readonly", "bool", false),
-			"volume_options": hclspec.NewBlockSet("volume_options", hclspec.NewObject(map[string]*hclspec.Spec{
```

Comment:
Not related to PR - but noticed that `volume_options` is expected to be a single block not a collection.